CHANGELOG-hmp-97-add-download-notebook-to-expanded-view.md

```diff
@@ -0,0 +1 @@
+- Add Jupyter Notebook download button to expanded visualization view
```

Comment:
```suggestion
- Add Jupyter Notebook download button to expanded visualization view.
```

Reply:
Sorry to be annoying!

Reply:
Not annoying at all, I appreciate the correction when I miss details like this 👍